Dockerfile

```diff
@@ -1,9 +1,9 @@
 FROM python:3.8.10
```

Comment:
Do we want to take the opportunity to update to 3.10 or 3.11 since we're making a bunch of other major changes?